README.md

```diff
@@ -1,3 +1,10 @@
 # terraform-drupal
 
 Repository that hosts terraform code to launch drupal/magento setup in AWS cloud
+
+# Setting up the environment 
+How to setup the environment </br>
+
+Copy the .env-template content and make a file on your local system with .env name. </br>
+Edit the access key and access ID accordingly. Also, edit the AWS region to us-east-1. </br>
+To check whether the changes are correct, run the command "make plan". </br>
```

Comment:
How to setup local environment

Mention that the .env file should be in same directory. 
Access Key and Secret Access Key Id. Region they can choose as per them